corrcal/corrcal.py

```diff
@@ -1,141 +1,100 @@
-import ctypes
 import time
-import site
-from glob import glob
 
 import numpy as np
 
+from .wrapper import *
```

Comment:
We should get rid of this import *

Reply:
Fixed in https://github.com/UKZN-Astronomy/corrcal/pull/21/commits/635a6eed3f47cf2b2155b6c109869b9f4f8ffc0a